README.md

```diff
@@ -5,3 +5,13 @@
 1. `bower install`
 2. `npm install`
 3. `npm run build`
+
+## Preparing a tag to release
+
+To prepare a tag to release you have to run the following command from the root directory of bundle:
```

Comment:
To prepare a tag for release you have to run the following command from the root directory of the bundle: